gz3d/src/gziface.js

```diff
@@ -960,6 +960,13 @@ GZ3D.GZIface.prototype.createGeom = function(geom, material, parent)
         if (idx > 1)
         {
           modelName = meshUri.substring(meshUri.lastIndexOf('/', idx-1));
+        } else {
+          console.log('Could not establish a local filename for mesh: "' +
+            meshUri +
+            '". Mesh URIs using an absolute path syntax should use ' +
+            'the ROS-standard directory structure of ' +
+            '"<package_path>/meshes/<mesh_path>", e.g. ' +
+            '"/home/user/workspace/src/ur_description/meshes/base.dae".');
```

Comment:
GzWeb is not ROS-specific, do you mind rephrasing the warning so it doesn't mention ROS? I know that the comment above mentions `gazebo_ros_pkgs` as an example of how a user may end up here, but there are other ways in which absolute paths could be passed to Gazebo too.

How about something like:

"Mesh URIs using an absolute path syntax should be located inside a `meshes` directory, e.g. /home/user/workspace/src/ur_description/meshes/base.dae"